src/component/registry/bpmn-elements-registry.ts

```diff
@@ -123,6 +129,18 @@ export class BpmnElementsRegistry {
    * bpmnVisualization.bpmnElementsRegistry.addCssClasses('task_3', ['suspicious-path', 'additional-info']);
    * ```
    *
+   * **Notes**:
+   *
+   * This method is intended to set CSS classes to specific elements, for instance to hide or highlight them. During BPMN diagram rendering, `bpmn-visualization` set specific CSS classes to all elements regarding their types.
+   * So, if you want to style all elements of a given type, use this default classes instead of adding new ones. The classes allow identifying elements of the same 'family' and of the same specific type.
```

Comment:
```suggestion
   * So, if you want to style all elements of a given type, use these default classes instead of adding new ones. The classes allow identifying elements of the same 'family' and of the same specific type.
```